src/plugins/intel_gpu/include/intel_gpu/primitives/prior_box8.hpp

```diff
@@ -0,0 +1,167 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+#pragma once
+
+#include <cmath>
+#include <limits>
+#include <vector>
+
+#include "primitive.hpp"
+
+namespace cldnn {
+/// @addtogroup cpp_api C++ API
+/// @{
+/// @addtogroup cpp_topology Network Topology
+/// @{
+/// @addtogroup cpp_primitives Primitives
+/// @{
+
+/// @brief Generates a set of default bounding boxes with different sizes and aspect ratios.
+/// @details The prior-boxes are shared across all the images in a batch (since they have the same width and height).
+/// First feature stores the mean of each prior coordinate.
+/// Second feature stores the variance of each prior coordinate.
+struct prior_box8 : public primitive_base<prior_box8> {
+    struct Attributes {
+        std::vector<float> min_size;       // Desired min_size of prior boxes
+        std::vector<float> max_size;       // Desired max_size of prior boxes
+        std::vector<float> aspect_ratio;   // Aspect ratios of prior boxes
+        std::vector<float> density;        // This is the square root of the number of boxes of each type
+        std::vector<float> fixed_ratio;    // This is an aspect ratio of a box
+        std::vector<float> fixed_size;     // This is an initial box size (in pixels)
+        bool clip;                         // Clip output to [0,1]
+        bool flip;                         // Flip aspect ratios
+        float step;                        // Distance between prior box centers
+        float offset;                      // Box offset relative to top center of image
+        std::vector<float> variance;       // Values to adjust prior boxes with
+        bool scale_all_sizes;              // Scale all sizes
+        bool min_max_aspect_ratios_order;  // Order of output prior box
+    };
+
+    CLDNN_DECLARE_PRIMITIVE(prior_box8)
+    prior_box8(const primitive_id& id,
+               const std::vector<primitive_id>& inputs,
+               int32_t height,
+               int32_t width,
+               int32_t image_height,
+               int32_t image_width,
+               Attributes attributes,
+               const cldnn::data_types output_type)
+        : primitive_base{id, inputs, ext_prim_id, padding(), optional_data_type(output_type)},
+          attributes{attributes},
+          width{width},
+          height{height} {
+        const int64_t numbers = number_of_priors(attributes);
+        output_shape = get_output_shape(height, width, numbers);
+        aspect_ratios = {1.0f};
+        for (const auto& aspect_ratio : attributes.aspect_ratio) {
+            bool exist = false;
+            for (const auto existed_value : aspect_ratios) {
+                exist |= std::fabs(aspect_ratio - existed_value) < 1e-6;
```

Comment:
Name for 1e-6 ?

Reply:
It looks like tolerance but I'm not sure to do it as separated constant